### PR TITLE
update checks doc

### PR DESCRIPTION
**Pull request by hathin**
- CA check, security default, and service account
+ co-management

@tiaraquan  FYI

### CODE REVIEW

File: windows/deployment/windows-autopatch/prepare/windows-autopatch-fix-issues.md [hunk 1]
```diff
@@ -55,14 +55,14 @@ Your "Windows 10 update ring" policy in Intune must not target any Windows Autop
 
 You can access Azure Active Directory (AD) settings in the [Azure portal](https://portal.azure.com/).
 
-### Conditional access policies
+### Co-Management
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopatch/prepare/windows-autopatch-fix-issues.md:58]:
```suggestion
### Co-management
```

File: windows/deployment/windows-autopatch/prepare/windows-autopatch-fix-issues.md [hunk 2]
```diff
@@ -55,14 +55,14 @@ Your "Windows 10 update ring" policy in Intune must not target any Windows Autop
...
-| Error | The Intune Administrator  role doesn't have sufficient permissions for this check. You'll also need to have these Azure Active Directory (AD) roles assigned to run this check:<br><ul><li>Security Reader</li><li>Security Administrator</li><li>Conditional Access Administrator</li><li>Global Reader</li><li>Devices Administrator</li></ul> |
+| Advisory | To successfully enroll devices that are Co-Managed with Windows Autopatch, it is necessary that the following Co-Managed workloads are switched to Intune.
+:<ul><li>Device Configuration</li><li>Windows Update policies</li><li>Office 365 Client Apps</li></ul><p>Should co-management not apply to your tenant, this check can be safely disregarded and will not block deployment of devices.</p> |
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopatch/prepare/windows-autopatch-fix-issues.md:65]:
```suggestion
| Advisory | To successfully enroll devices that are co-managed with Windows Autopatch, the following co-managed workloads should be switched to Intune:<br><ul><li>Device Configuration</li><li>Windows update policies</li><li>Office 365 client apps</li></ul><p>Should co-management not apply to your tenant, this check can be safely disregarded and will not block the deployment of devices.</p> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hathin, kindly commit the suggested changes. Many thanks.

**Comment by hathin** [comment]
@JohanFreelancer9 committed the changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiaraquan** [comment]
As per discussion, closing this PR. Changes were consolidated in PR #6978.

**Comment by tiaraquan** [comment]
#sign-off

**Pull request closed without merging**